hibernate-core/src/main/java/org/hibernate/cfg/annotations/reflection/internal/JPAXMLOverriddenAnnotationReader.java

```diff
@@ -251,10 +251,10 @@
 		annotationToXml.put( AccessType.class, "access" );
 		annotationToXml.put( AttributeOverride.class, "attribute-override" );
 		annotationToXml.put( AttributeOverrides.class, "attribute-override" );
-		annotationToXml.put( AttributeOverride.class, "association-override" );
-		annotationToXml.put( AttributeOverrides.class, "association-override" );
-		annotationToXml.put( AttributeOverride.class, "map-key-attribute-override" );
-		annotationToXml.put( AttributeOverrides.class, "map-key-attribute-override" );
+		annotationToXml.put( AssociationOverride.class, "association-override" );
+		annotationToXml.put( AssociationOverrides.class, "association-override" );
+		//annotationToXml.put( AttributeOverride.class, "map-key-attribute-override" );
+		//annotationToXml.put( AttributeOverrides.class, "map-key-attribute-override" );
```

Comment:
Not sure what we can do for the above two annotations

Reply:
AFAICT the code only uses `annotationToXml.containsKey()` and once `annotationToXml.getOrDefault()` for exception reporting, so this is fine IMO.